src/ReadVTK.jl

```diff
@@ -1132,4 +1140,27 @@ function get_example_file(filename; head="main", output_directory=".", force=fal
   return filepath
 end
 
+"""
+    convert_VTKCells_to_MeshCells(cells::VTKCells)
+
+Converts the `VTKCells` type, which holds data on N number of cells, to
+a vector of N number of `MeshCell` types. Allows for the reading and then
+subsequent writing with `WriteVTK`.
```

Comment:
```suggestion
Convert a `VTKCells` object, which holds raw point and connectivity data for a number of cells, to
a vector of `MeshCell` objects. The latter can, e.g., be passed to the WriteVTK.jl package for writing new VTK files.
```
We are not really converting types but objects of a given type; I hope this makes the comment a little clearer.